mods/ra/rules/defaults.yaml

```diff
@@ -626,17 +626,15 @@
 		RequiresCondition: cruising
 	BodyOrientation:
 		UseClassicFacingFudge: True
-
```

Comment:
These empty lines (a lot of them also removed in other places) should be reverted as they were for YAML readability (they split the code blocks).